LCM/scripts/python3/StartDscConfiguration.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+# Standard library imports
+from subprocess           import Popen, PIPE
+from sys                  import argv
+from imp                  import load_source
+from os.path              import dirname, isfile, join, realpath
+from fcntl                import flock, LOCK_EX, LOCK_UN, LOCK_NB
+from OmsConfigHostHelpers import write_omsconfig_host_telemetry, write_omsconfig_host_switch_event, write_omsconfig_host_log, stop_old_host_instances
+from time                 import sleep
+
+pathToCurrentScript = realpath(__file__)
+pathToCommonScriptsFolder = dirname(pathToCurrentScript)
+
+helperLibPath = join(pathToCommonScriptsFolder, 'helperlib.py')
+helperlib = load_source('helperlib', helperLibPath)
+
+
+def main(argv):
+    """StartDscConfiguration"""
+
+    # Define method arguments and description
+    description = 'Starts the specified DSC configuration.'
+
+    parameters = {
+        'configurationmof' : {
+            'shortForm' : 'c',
+            'helpText' : 'The path to the configuration mof to start.',
+            'required' : True,
+            'action' : 'store'
+        },
+        'force' : {
+            'shortForm' : 'f',
+            'helpText' : 'Specifies that any current pending configuration should be forcibly removed before starting the new configuration.',
+            'required' : False,
+            'action' : 'store_true'
+        }
+    }
+
+    # Parse -configurationmof on its own for backwards compatibility
+    configmofArgument = None
+    if '-configurationmof' in argv:
+        configmofIndex = argv.index('-configurationmof')
+
+        try:
+            configmofArgument = argv[configmofIndex + 1]
+        except:
+            print 'StartDscConfiguration.py: error: Please provide a valid path argument for -configurationmof'
```

Comment:
I think these might pop up an error in python3. There are a few places where this is not taken care.